dev-php/php-meminfo/php-meminfo-1.1.1.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+USE_PHP="php7-4"
```

Comment:
Is this right? What about an of the newer version? The Github repo says it does also support 8.0.

Reply:
To be honest: I'm not hundred percent sure. It does definitely not build with php 8.1
And for PHP 8.0 i have currently no ebuild available to build, because there is no ssl-compat-based build in php-legacy yet.

Reply:
Ok. I was just wondering. If anyone needs this for any other version, they might have to add an issue.

Reply:
It's not "that" bad. 
This commit already includes a "prebuild" 2.0.0 (not yet tagged, but prepared by upstream) ebuild, which builds against 7.4 - 8.1, 8.2 and 8.3 (verified ;) )